### DISCUSSION TITLE
Correct usage of network key and ext_pan_id for migration

### DISCUSSION THREAD

**Discussion by gnunzi** [Koenkk/zigbee2mqtt#7905] [General]
Hi everyone,

I used so far OpenHab with the integrated Zigbee addon and I would like to use zigbee2mqtt without re-pairing all the devices.
Unfortunately the existing devices are not recognized by zigbee2mqtt, so I assume I am doing something wrong with the network key or the ext_pan_id.
In OpenHab I have the following net security key: `50269A21D27A6B87ACC0xxxxxxxxxxxx`
I tried to convert the network_key  to decimal as follow:
`network_key: [80,38,154,33,210,122,107,135,172,192,xx,xx,xx,xx,xx,xx]`
In OpenHab I had the following extended pan ID: `D13528441Cxxxxxx`
I converted it as follows:
` ext_pan_id: [0xD1,0x35,0x28,0x44,0x1C,0xxx,0xxx,0xxx]`

The channel and the pan ID are the same.

Why are the existing devices not recognized? Am I doing some error in defining the network key or ext_pan_id?

Thanks for any help!

**Comment by Koenkk** [comment]
When switching from a different zigbee system to Zigbee2MQTT you need to repair all devices.
